rollbar/__init__.py

```diff
@@ -548,6 +554,17 @@ def prev_page(self):
         return _get_api(self.path, endpoint=self.endpoint, **params)
 
 
+def DRF_post_exception_handler(exc, context):
```

Comment:
I think this should probably go in `contrib`, like the other framework-specific handlers: https://github.com/rollbar/pyrollbar/tree/master/rollbar/contrib